### PR TITLE
Update to superset 4.0.0

### PR DESCRIPTION
**Pull request by vivian-rook**
Bug: T364022

### CODE REVIEW

File: ansible/files/csi-secret-cinderplugin.yaml.codfw1dev.crypt [hunk 1]
```diff
@@ -7,4 +7,4 @@ metadata:
   name: cloud-config
   namespace: kube-system
 data:
```
Comment (Gadberry3) [inline ansible/files/csi-secret-cinderplugin.yaml.codfw1dev.crypt:9]: Suck

File: ansible/files/csi-secret-cinderplugin.yaml.codfw1dev.crypt [hunk 2]
```diff
@@ -7,4 +7,4 @@ metadata:
...
-  cloud.conf: W0dsb2JhbF0KYXBwbGljYXRpb24tY3JlZGVudGlhbC1pZCA9IGQ3NWI0MmE0NWVhYTQ4ZWE5MjhhMzE2NDg0ZTI1NGI0CmFwcGxpY2F0aW9uLWNyZWRlbnRpYWwtc2VjcmV0ID0gdHNDeGZfclV1RmVWY2tIOV9OMjEwRF9fZDZUZWFoSEdwWVQ0NEIwT0MyT29xdEdxV292QktuclRta3U2aDZqVjN4eF81ZUdndkV6UEtQSkpyM3dvRncKZG9tYWluLW5hbWUgPSBkZWZhdWx0CmF1dGgtdXJsID0gaHR0cHM6Ly9vcGVuc3RhY2suY29kZncxZGV2Lndpa2ltZWRpYWNsb3VkLm9yZzoyNTAwMC92Mwp0ZW5hbnQtaWQgPSBrOHMtZGV2CnJlZ2lvbiA9IGNvZGZ3MWRldi1yCg==
```
Comment (Gadberry3) [inline ansible/files/csi-secret-cinderplugin.yaml.codfw1dev.crypt:10]: Ok

Comment (Gadberry3) [inline ansible/files/csi-secret-cinderplugin.yaml.codfw1dev.crypt:10]: Ok

**Review by Gadberry3 — APPROVED** [review]

**Comment by andrewbogott** [comment]
Merging because this seems to be already deployed :)

**Comment by vivian-rook** [comment]
This one wasn't done. There's more discussion in the ticket about the encountered error https://phabricator.wikimedia.org/T364022

**Pull request merged**